charts/aiven-operator-crds/templates/aiven.io_postgresqls.yaml

```diff
@@ -637,6 +637,7 @@ spec:
                     - "13"
                     - "14"
                     - "15"
+                    - "10"
```

Comment:
this has been [deprecated](https://github.com/aiven/go-api-schemas/blob/main/pkg/dist/service_types.yml#L4290-L4292) 🤔

Reply:
So thought I, but then checked out the source json: 
<img width="283" alt="image" src="https://user-images.githubusercontent.com/1560043/229067877-c02af48b-8cbc-4663-90a8-e471fc2dbbed.png">

Reply:
Please double check this. It looks weird.

Reply:
<img width="389" alt="image" src="https://user-images.githubusercontent.com/1560043/229068791-30fabeb6-f88c-452f-b05b-b60eb207b9d1.png">
It is available through the console. And also come out too.

Reply:
"coming out of support" means it's deprecated

Reply:
there is no v10 in the source JSON:

```json
{
   "pg_version":{
      "title":"PostgreSQL major version",
      "type":[
         "string",
         "null"
      ],
      "enum":[
         "11",
         "12",
         "13",
         "14",
         "15"
      ]
   }
}
```